test/common/dub.sdl

```diff
@@ -0,0 +1,5 @@
+name "common"
+description "utility package for test suit"
+authors "drug007"
+copyright "The D foundation"
+license "BSL"
```

Comment:
```suggestion
license "BSL-1.0"
```

For software licenses, you usually want to use the SPDX identifier: https://spdx.org/licenses/BSL-1.0.html